tests/markdown_list/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -5,9 +5,9 @@ exports[`checkbox.md 1`] = `
 - [x] this is a long long long long long long long long long long long long long long paragraph.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * [ ] this is a long long long long long long long long long long long long long
-      long paragraph.
+  long paragraph.
```

Comment:
This is kinda undesirable; can we keep the alignment for non-checked list items?